server/publicapi/tests/items_service_test.py

```diff
@@ -187,7 +187,7 @@ def test_sets_fields_filter_on_request_object(self, fake_set_fields_filter):
         instance.get(fake_request, lookup)
 
         self.assertTrue(fake_set_fields_filter.called)
-        args, kwargs = fake_set_fields_filter.call_args
+        args, _kwargs = fake_set_fields_filter.call_args
```

Comment:
If the intent of the renaming is to explicitly mark the `kwargs` variable as unused, it's probably better to simply use a single underscore for that (an often used convention in Python).